src/realm/table.hpp

```diff
@@ -110,6 +110,9 @@ class Table {
     /// string.
     StringData get_name() const noexcept;
 
+    // Get table name with class prefix removed
+    StringData get_class_name() const noexcept;
```

Comment:
Backported from the exception unification branch. This should probably be used in all error messages.